### PR TITLE
Find mongodb odm

### PR DESCRIPTION
**Pull request by CristiPV**
Found a MongoDB ODM: [Beanie](https://roman-right.github.io/beanie/), and implemented basic CRUD endpoints for the wallet model to test it out. If you like the way it works and you agree with the implementation, approve the PR, otherwise, let me know if I should try something else.

### CODE REVIEW

File: backend/app/models/wallet.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+from typing import Optional
+
+from beanie import Document
+
+
+class Wallet(Document):
+    nickname: str
+    balance: Optional[int] = 0
```
Comment (ithai5) [inline backend/app/models/wallet.py]: I guess that optional can be null? 
if yes I think that there always should be value. 
is `int` is the same as `int` in Java?  It should be able to hold decimal number

File: backend/app/routers/test_wallets.py [hunk 2]
```diff
@@ -5,7 +5,10 @@
 client = TestClient(app)
 
 
-def test_read_wallets():
```
Comment (ithai5) [inline backend/app/routers/test_wallets.py:8]: I guess that we will need to find out how to test the endpoint then

**Review by ithai5 — APPROVED** [review]

File: backend/app/models/wallet.py (hunk 1 above)
Comment (CristiPV) [reply backend/app/models/wallet.py]: I will test that out then and see how it behaves.

File: backend/app/routers/test_wallets.py (hunk 2 above)
Comment (CristiPV) [reply backend/app/routers/test_wallets.py:8]: Yeah, it is what I wanted to discuss. We can either have tests along with the functionality or do the whole backend and then test it. I would have good reasons for going with each approach. On second thought tho, I think I would prefer not to pile up a lot of testing at the end, so I will try to find a way to test everything for this one, then merge it, and we keep testing as we implement things.

File: backend/app/routers/wallets.py [hunk 3]
```diff
@@ -1,10 +1,46 @@
...
 
 
-@router.get("/")
+@router.get("/", response_model=List[Wallet])
 async def read_wallets():
-    return {"response": "WALLETS !!!"}
+    return await Wallet.find_all().to_list()
+
+
+@router.get("/{wallet_id}", response_model=Wallet)
```
Comment (ithai5) [inline backend/app/routers/wallets.py:18]: I think that we are missing some logic regarding the relation between the user and the wallet. Like user can only view his own wallet.

File: backend/app/main.py [hunk 4]
```diff
@@ -1,11 +1,24 @@
...
+@lru_cache()
+def get_settings():
+    return Settings()
+
+
+@app.on_event("startup")
+async def init():
+    await init_db(get_settings())
+
 app.include_router(wallets.router)
```
Comment (ithai5) [inline backend/app/main.py:22]: I guess that middleware we will add at a later point?

**Review by ithai5 — APPROVED** [review]
It looks good, I really like the test and how you figured it out. I think that it will be a good idea to divide the responsibility of the router and the database communication.

**Pull request merged**